client/js/views/GameView.vue

```diff
@@ -645,20 +648,13 @@ export default {
       this.targeting = false;
     },
     clearSelection() {
-      this.selectionIndex = null;
+      this.selectCard(null);
       this.secondCardIsSelected = false;
       this.topCardIsSelected = false;
       this.targetingMoveName = null;
       this.targetingMoveDisplayName = null;
       this.clearOverlays();
     },
-    selectCard(index) {
-      if (index === this.selectionIndex) {
-        this.clearSelection();
-      } else {
-        this.selectionIndex = index;
-      }
-    },
```

Comment:
Changing this has unintended side effects. It breaks card targeting as those work by index (see `targetOpponentPointCard` and `targetOpponentFaceCard`). There's also another unintended side effect where the card overlay pops right before trying to target a card and I'm certain it's due to the order of operations.

I likely need to reconfigure the logic in the card overlay to know when it is and is not valid for showing or hiding now that we're saying it can show anytime. There are certain states in the game we'll need to track more transparently. cc @itsalaidbacklife